tests/ui/perfherder/tests_table_test.jsx

```diff
@@ -61,3 +61,25 @@ test('Tests table should show data', async () => {
   expect(result1).toBeInTheDocument();
   expect(result2).toBeInTheDocument();
 });
+
+test('Clicking on platform icon displays the list of platforms', async () => {
+  const { getAllByTestId, getByTestId } = testsTable(
+    results,
+    projectsMap,
+    platformsMap,
+  );
+
+  const platformIcon = await waitFor(() => getAllByTestId('other'));
```

Comment:
I not a fan of `data-testid="other"`, perhaps we could change it to `data-testid="otherPlatforms"` or something similar.